### ISSUE TITLE
after update to HA OS 9.6 wrong display

### ISSUE THREAD

**Issue by maxnomo60** [Yevgenium/weather-chart-card#119]
Hi, after update from Homeassitant OS 8.xx to 9.xx (now 9.7) cards do no more show the forecast data like attached pic.  Any idea ?
![weather](https://user-images.githubusercontent.com/83511993/193248938-665f18ed-f17e-4f2d-9381-b856b8c2616e.JPG)

**Comment by maxnomo60** [comment]
I now found out hat on my Android Mobile App the Graphs with temeprature are showing up perfectly. The same with Internet Browser EDGE from Microsoft.  It seems to be a problem with FIREFOX !!

**Comment by greendog99** [comment]
I updated the version of [chart.js ](https://www.chartjs.org) from 3.5.1 to 3.9.1 in the weather-chart-card and now it renders correctly in Firefox. I've submitted a pull request, but in the meantime you can get my fixed version at https://github.com/greendog99/weather-chart-card/releases

**Comment by boheme61** [comment]
> I updated the version of [chart.js ](https://www.chartjs.org) from 3.5.1 to 3.9.1 in the weather-chart-card and now it renders correctly in Firefox.

Thx alot , just " downloaded " your fork, instead :) , im a "Notorious" FF user, so i've been pretty annoyed lately, until i decided go for your fork ,

**Comment by tomlut** [comment]
It would appear this repository has been abandoned.

**Comment by boheme61** [comment]
Yes that was also my thought, so i've actually spend a few days looking into other weather-cards(before i "paused" it, and get this "temporary" fix ( thou it still seems like FF has problems with the rendering)
I would probably go back to  native-ha, if i somehow could get a good looking "precipitation" overview for coming 5 days ( and hourly ofcause )
